admin/python/lsst/qserv/admin/nodePool.py

```diff
@@ -129,7 +128,7 @@ def success(p):
                 process_id += 1
                 node = remaining_nodes.pop()
 
-                task = node.getSshCmd(command);
+                task = node.getCommand(command)
```

Comment:
ok I'll do it, FYI it"s (briefly) documented here: http://www.slac.stanford.edu/exp/lsst/qserv/2015_05/HOW-TO/cluster-deployment.html#deploy-on-the-cluster